HelpSource/Classes/ServerTreeView.schelp

```diff
@@ -0,0 +1,55 @@
+class:: ServerTreeView
+summary:: Show Server-side Node Tree
+categories:: GUI>Views
+related:: Classes/ServerTreeView
+
+description::
+ServerTreeView shows the node tree on a server.
+
+note::
+The link::Classes/ServerTreeView:: will disappear when the pointed server is not running.
+::
+
+classmethods::
+
+method:: new
+argument:: aServer
+The server to display its node tree.
+argument:: bounds
+A link::Classes/Rect::, or any object responding to the link::Search#asRect:: method.
+argument:: interval
+Polling interval.
+
+method:: closeAll
+Close all instances of link::Classes/ServerTreeView::
+
+instancemethods::
+
+method::close
+Close the receiver.
+
+examples::
+
+code::
+// Start server
+s.boot;
+
+// Create a ServerTreeView:
+a = ServerTreeView.new(s)
```

Comment:
either both with `.new` or none.

Reply:
OK, Done! `.new' for both.
https://github.com/supercollider/supercollider/pull/6294/commits/1bc7706c859daafa68f26caf704e501f8ad8c5d4